vendor/github.com/aws/aws-sdk-go/aws/awsutil/prettify.go

```diff
@@ -1,113 +0,0 @@
-package awsutil
```

Comment:
I would actually love for this to not be removed unless there's actually a need. It's incredibly useful for being able to print structs and inspect contents in a human readable way.